components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -101,7 +100,6 @@ export class TeamDBImpl extends TransactionalDBImpl<TeamDB> implements TeamDB {
             return {
                 userId: u.id,
                 fullName: u.fullName || u.name,
-                primaryEmail: User.getPrimaryEmail(u),
```

Comment:
@AlexTugarev  spec.ts file not updated so main build was failed.

Is it safe to remove? Since there're lots of places will use `findMembersByTeam` function